blazor/datagrid/scrolling.md

```diff
@@ -511,6 +511,92 @@ The [GridColumns](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.
 
 ![Add or Remove Frozen Blazor DataGrid Columns by Dragging the Column Separator](./images/blazor-datagrid-drag-and-drop-the-column-separator.gif)
 
+## Show or hide the column separator
+
+By default, enabling the [AllowFreezeLineMoving](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.SfGrid-1.html#Syncfusion_Blazor_Grids_SfGrid_1_AllowFreezeLineMoving) property of the DataGrid component, the frozen column separator will be displayed at the left and right end of the DataGrid component. You can show or hide this column separator using the below CSS style.
```

Comment:
what is the need of this requirement. Use case is not understandable. And it is not correct to say that the frozen line is displayed in right and left of the datagrid when we simply set this AllowFreezeLinemoving is true .

Reply:
support team logged feature related to this topic. so for now we can hold this topic.

Ticket details: https://support.syncfusion.com/agent/tickets/392894